build/Linux+BSD/portable/copy-libs

```diff
@@ -72,7 +74,7 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sensors.so.5
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
-  copyLib libQt5WebChannel.so.5
```

Comment:
but the thing is musescore AppImage seems to run just fine without libQt5WebChannel.so.5 after I unstalled libQt5WebChannel and rebooted.  So it seems to be OK...